src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -94,6 +98,13 @@
 
     public final StreetNotesService streetNotesService = new StreetNotesService();
 
+    /**
+     * Allows a notice element to be attached to an object in the OTP model and then retrieved
+     * by the API when navigating from that object. This requires equals and hashCode to be
+     * implemented properly.
+     */
+    private final Multimap<NoticeAssignable, Notice> noticesByElement = HashMultimap.create();
+
     // transit feed validity information in seconds since epoch
```

Comment:
Do noticesByElement need to be part of both Graph and Index - isn´t it enough to include it in the index. 

Naming: Should be: `noticeAssignment`

Reply:
Removed from Index.